admin_ui/app/App.js

```diff
@@ -1,60 +1,30 @@
 import React from 'react';
-import Loadable from 'react-loadable';
 import { Router } from '@reach/router';
 import './main.css';
-import LoadingIndicator from '../components/LoadingIndicator';
 import Store from './Store';
 
-const AsyncLogin = Loadable({
-  loader: () => import(/* webpackChunkName: "login" */ './session/LoginPage'),
-  loading: LoadingIndicator,
-  delay: 300, // 0.3 seconds
-});
-
-const AsyncDashboard = Loadable({
-  loader: () => import(/* webpackChunkName: "dashboard" */ './dashboard/DashboardPage'),
-  loading: LoadingIndicator,
-  delay: 300, // 0.3 seconds
-});
-
-const AsyncOrganization = Loadable({
-  loader: () => import(/* webpackChunkName: "org" */ './org/OrgPage'),
-  loading: LoadingIndicator,
-  delay: 300, // 0.3 seconds
-});
-
-const AsyncPermission = Loadable({
-  loader: () => import(/* webpackChunkName: "permission" */ './permission/PermissionPage'),
-  loading: LoadingIndicator,
-  delay: 300, // 0.3 seconds
-});
+/*
+  We want both the Login view and the Dashboard view to be part of the main bundle
+  (and thus appear immediatelly to the user once the bundle is loaded)
+  DashboardSection contains a nested <Router> which loads all
+  other pages using react-loadable
+ */
+import LoginPage from './session/LoginPage';
+import DashboardSection from './dashboard/DashboardSection';
+import ForgotPasswordPage from './session/ForgotPasswordPage';
 
-const AsyncRole = Loadable({
-  loader: () => import(/* webpackChunkName: "role" */ './role/RolePage'),
-  loading: LoadingIndicator,
-  delay: 300, // 0.3 seconds
-});
 
-const AsyncUser = Loadable({
-  loader: () => import(/* webpackChunkName: "user" */ './user/UserPage'),
-  loading: LoadingIndicator,
-  delay: 300, // 0.3 seconds
-});
 
 /**
  * The top level component / wrapper
- * encompassing the Header, Nav and PageWrapper
  */
 const App = () => {
   return (
     <Store.Provider>
       <Router>
-        <AsyncLogin path="/login" />
-        <AsyncDashboard path="/" />
-        <AsyncOrganization path="/organizations/*" />
-        <AsyncPermission path="/permissions/*" />
-        <AsyncRole path="/roles/*" />
-        <AsyncUser path="/users/*" />
+        <DashboardSection path="/*" />
```

Comment:
_crazy_ important here to have `/*` instead of `/` otherwise the nested `<Router>` would not work! Spend 30 mins on this stupid thing....grrrr